environments.json

```diff
@@ -1265,7 +1265,7 @@
       "non-standard"
     ]
   },
-  "safari51": {
```

Comment:
I believe both would be about the same amount of work.  But I pushed my patch first... :)